cdap-docs/build.sh

```diff
@@ -460,18 +490,105 @@ function copy_docs_inner_level() {
 }
 
 function build_zip() {
+  local zip_target=${1}
+  local zip_extras=${2}
   set_project_path
-  make_zip ${1}
-}
-
-function zip_extras() {
-  if [[ "x${1}" == "x" ]]; then
-    return
+  set_version
+  if [ "x${1}" == "x" ]; then
+    ZIP_DIR_NAME="${PROJECT}-docs-${PROJECT_VERSION}"
+  else
+    ZIP_DIR_NAME="${PROJECT}-docs-${PROJECT_VERSION}-$1"
   fi
-  echo "Adding .htaccess file (404 file)"
-  rewrite ${SCRIPT_PATH}/${COMMON_SOURCE}/${HTACCESS} ${TARGET_PATH}/${PROJECT_VERSION}/.${HTACCESS} "<version>" "${PROJECT_VERSION}"
   cd ${TARGET_PATH}
-  zip -qr ${ZIP_DIR_NAME}.zip ${PROJECT_VERSION}/.${HTACCESS}
+  doc_change_py="${TARGET_PATH}/../tools/doc-change.py"
+
+  echo "Removing old directories and zips"
+  rm -rf base current ${PROJECT_VERSION} *.zip
+  local errors=$?
+  if [ "${errors}" != "0" ]; then
+      echo "Could not cleanup old directories and zips"
+      return ${errors}   
+  fi
```

Comment:
If you're using `-f` with `rm`, it'll always return 0, even on failure. There's no need to check the exit on this.